hipify-clang/src/ArgParse.cpp

```diff
@@ -85,8 +85,22 @@ cl::opt<bool> Examine("examine",
   cl::cat(ToolTemplateCategory));
 
 cl::opt<bool> DashDash("-",
-  cl::desc("Separator between hipify-clang and clang options;\ndon't specify if there are no clang options."),
+  cl::desc("Separator between hipify-clang and clang options;\ndon't specify if there are no clang options"),
   cl::value_desc("--"),
   cl::cat(ToolTemplateCategory));
 
+cl::list<std::string> I("I",
+  cl::desc("Add directory to include search path"),
+  cl::value_desc("directory"),
+  cl::ZeroOrMore,
+  cl::Prefix,
+  cl::cat(ToolTemplateCategory));
+
+cl::list<std::string> D("D",
+  cl::desc("Define <macro> to <value> or 1 if <value> omitted"),
+  cl::value_desc("macro>=<value"),
```

Comment:
typo?

Reply:
`macro>=<value` ? No, it is not, outter angle brackets will be substituted by clang tooling - it might be seen in hipify-clang output by setting '-help' option.